workflows/proteomics/clinicalmp/clinicalmp-quantitation/iwc-clinicalmp-quantitation.ga

```diff
@@ -0,0 +1,404 @@
+{
+    "a_galaxy_workflow": "true",
+    "annotation": "Clinical Metaproteomics ",
+    "comments": [],
+    "creator": [
+        {
+            "class": "Organization",
+            "name": "GalaxyP"
```

Comment:
You could also add yourself and your orcid to the creators, and maybe a URL for the GalaxyP organization >

Reply:
The galaxyP url is not secure so I wasn't sure if I should add it

Reply:
Also, I cant change the run time as these files are important for the run, to show how the workflow works

Reply:
The test files are not meant for demonstrating how the workflow works, it is meant for us to know that all the pieces work together correctly. You can describe larger test data in the readme or the GTN if you like.